tests/rptest/test_suite_ec2.yml

```diff
@@ -4,17 +4,12 @@ ec2:
     - tests
 
   excluded:
-    - tests/retention_policy_test.py # Redpanda node failed to stop in 30 second
```

Comment:
Thanks for updating this. As you can see it was a bit outdated based on Noah's original prototype.